### ISSUE TITLE
CAT can't work in Diego

### ISSUE THREAD

**Issue by kei-yamazaki** [cloudfoundry/cf-release#857]
I deployed cf-release v225 and diego-release v0.1441.0 on OpenStack via BOSH.
This environment is only deployed garden-linux, not deployed DEAng.

After deploy, I execute CAT but it's failed because ```cf app``` command doesn't work.
Following is manually execution logs of ```cf app``` command in CAT's VM.
```
root@c42d6a3c-d909-489e-9022-f8af4e9d3bb7:/var/vcap/bosh_ssh/bosh_d65h257d8# /var/vcap/packages/cli/bin/cf -v
/var/vcap/packages/cli/bin/cf version 6.12.1-56792aa-2015-07-06T22:48:00+00:00
root@c42d6a3c-d909-489e-9022-f8af4e9d3bb7:/var/vcap/bosh_ssh/bosh_d65h257d8# /var/vcap/packages/cli/bin/cf app dora1

...

REQUEST: [2015-11-20T01:33:16Z]
GET /v2/apps/71c49cd1-cfed-4b56-80d9-eed0c24bf923/stats HTTP/1.1
Host: api.10.0.50.100.tamac.me
Accept: application/json
Authorization: [PRIVATE DATA HIDDEN]
Content-Type: application/json
User-Agent: go-cli 6.12.1-56792aa / linux



RESPONSE: [2015-11-20T01:33:16Z]
HTTP/1.1 200 OK
Content-Length: 265
Content-Type: application/json;charset=utf-8
Date: Fri, 20 Nov 2015 01:33:16 GMT
Server: nginx
X-Cf-Requestid: fa468f79-d36c-4475-7428-7e1578357b84
X-Content-Type-Options: nosniff
X-Vcap-Request-Id: 1123a015-966f-4949-707d-ccc9adda7c97::1f28dda5-3cc1-4fb2-990e-558086e8717f

{"0":{"state":"RUNNING","stats":{"name":"dora1","uris":["dora1.10.0.50.100.tamac.me"],"host":"10.0.52.1","port":60264,"uptime":252,"mem_quota":268435456,"disk_quota":1073741824,"fds_quota":16384,"usage":{"time":"2015-11-20 01:33:16 UTC","cpu":0,"mem":0,"disk":0}}}}
FAILED
bad client request
FAILED
bad client request
```

I'm guessing this problem related [this ticket](https://www.pivotaltracker.com/n/projects/892938/stories/103051454).

Use latest version cf command, it's works.
```
root@c42d6a3c-d909-489e-9022-f8af4e9d3bb7:/var/vcap/bosh_ssh/bosh_d65h257d8# ./cf app dora1
Showing health and status for app dora1 in org admin-org / space default as admin...
OK

requested state: started
instances: 1/1
usage: 256M x 1 instances
urls: dora1.10.0.50.100.tamac.me
last uploaded: Fri Nov 20 01:28:39 UTC 2015
stack: cflinuxfs2
buildpack: ruby

     state     since                    cpu    memory      disk      details
#0   running   2015-11-20 01:29:03 AM   0.0%   0 of 256M   0 of 1G
```

**Comment by zaksoup** [comment]
@kei-yamazaki I just want to confirm, this is failing when you run the "acceptance-tests" errand against your deployment, but the `cf app` command that succeeds is being run from your local system?

Can you also provide your CF and Diego manifests (with any passwords or other private information redacted)?

Thanks!
@zaksoup & @njbennett

**Comment by kei-yamazaki** [comment]
@zaksoup Thank you for reply!

After that I checked my manifests and found my mistake, sorry.
This problem cause is wrong setting of the tps.

The tps can not communicate with the loggregator traffic controller, so tps can not get application metrics (such as cpu/memory/disk usages). 

And cli version 6.12.1-56792aa returns error when can not get application metrics but latest version cli zero value for metrics, does not return error (I think the preferred behavior of version 6.12).

I fixed tps settings and CAT works!

Now close the issue.
